### DISCUSSION TITLE
The Supabase project is not working in the panel or in the Javascript libraries

### DISCUSSION THREAD

**Discussion by jeanjanrley** [supabase/supabase#26350] [Questions]
From yesterday to today my project stopped working, I didn't receive any email informing me that my free quota limit had been exceeded.

I paused the project and it has been paused for more than two hours.

![WhatsApp Image 2024-05-15 at 07 10 49](https://github.com/supabase/supabase/assets/34939747/d0b7a042-4a00-439e-9991-27dbd4a18178)

**Comment by GaryAustin1** [comment]
Not sure there is an email, and not sure how that would freeze your project.

You say you paused the project.  Is it stuck pausing or restoring it back?

About all you can do is try a different browser and/or hard refresh your browser window.  If those don't work then you need to contact support.

**Comment by jeanjanrley** [comment]
It's stuck pausing.

![image](https://github.com/supabase/supabase/assets/34939747/12e2264a-ff30-4a7f-8b07-d31e0687d5c6)

**Reply by GaryAustin1** [reply]
If you did the different browser things then contact support.
